plugin/targetd/targetd.py

```diff
@@ -148,6 +152,20 @@ def plugin_register(self, uri, password, timeout, flags=0):
         self.headers = {'Content-Type': 'application/json',
                         'Authorization': 'Basic %s' % (auth,)}
 
+        if "no_ssl_verify" in self.uri["parameters"] \
+                and self.uri["parameters"]["no_ssl_verify"] == 'yes':
+            self.no_ssl_verify = True
+
+        if "cert_file" in self.uri["parameters"]:
+            # Check for file existence and throw error now if not present
+            self.cert_file = self.uri["parameters"]["cert_file"]
```

Comment:
Can we raise `INVALID_ARGUMENT` error for when both `no_ssl_verify` and `cert_file` is used?
`targetd+ssl://localhost?cert_file=<file_path>&no_ssl_verify=yes`